api-description/forms/pk/get.json

```diff
@@ -5,55 +5,66 @@
         "status": 200,
         "response_data": {
             "name": "How have you been using the site?",
-            "url": "http://localhost:8000/forms/1",
```

Comment:
Hmm you removed the `url` from this form, but that would be actually very useful for me